nautobot/dcim/tests/test_filters.py

```diff
@@ -1429,60 +1431,80 @@ class ManufacturerTestCase(FilterTestCases.NameSlugFilterTestCase):
     def setUpTestData(cls):
         common_test_data(cls)
 
-        devices = Device.objects.all()
-
-        manufacturers = (
-            Manufacturer.objects.create(name="Manufacturer 4", slug="manufacturer-4", description="A"),
-            Manufacturer.objects.create(name="Manufacturer 5", slug="manufacturer-5", description="B"),
-            Manufacturer.objects.create(name="Manufacturer 6", slug="manufacturer-6", description="C"),
-        )
+        devices = list(Device.objects.all()[:3])
 
-        InventoryItem.objects.create(device=devices[0], name="Inventory Item 1", manufacturer=manufacturers[0])
-        InventoryItem.objects.create(device=devices[1], name="Inventory Item 2", manufacturer=manufacturers[1])
-        InventoryItem.objects.create(device=devices[2], name="Inventory Item 3", manufacturer=manufacturers[2])
+        InventoryItem.objects.create(device=devices[0], name="Inventory Item 1", manufacturer=cls.manufacturers[0])
+        InventoryItem.objects.create(device=devices[1], name="Inventory Item 2", manufacturer=cls.manufacturers[1])
+        InventoryItem.objects.create(device=devices[2], name="Inventory Item 3", manufacturer=cls.manufacturers[2])
 
     def test_description(self):
-        params = {"description": ["A", "B"]}
-        self.assertEqual(self.filterset(params, self.queryset).qs.count(), 2)
+        # FIXME(jathan): We only want 1st/3rd manufacturer and only care right now because of random
+        # seed determinisim and mixed bag of static/dynamic fixture creation. Rip this out when all
+        # factories are done.
+        descriptions = [m.description for m in (self.manufacturers[0], self.manufacturers[-1])]
+        params = {"description": descriptions}
+        self.assertEqual(self.filterset(params, self.queryset).qs.count(), len(descriptions))
```

Comment:
This is returning blank descriptions randomly causing the tests to fail. Ex seed `m4O16QYOgFEoG9B0`

```suggestion
        manufacturers = self.queryset.exclude(description="")[:2]
        params = {"description": [manufacturers[0].description, manufacturers[1].description]}
        self.assertQuerysetEqualAndNotEmpty(
            self.filterset(params, self.queryset).qs, self.queryset.filter(description__in=params["description"])
        )
```

Reply:
Aren't we using a static seed for the tests at this time? Tests are passing in CI and locally!

Reply:
They are right now but may not when the Site/Region factories are merged in. Any time we update anything on a factory the randomness for all factories change

Reply:
Ah yes. So much fun. Thanks I'll merge in your change.